.github/workflows/scripts/e2e-verify.sh

```diff
@@ -143,7 +143,14 @@ e2e_verify_predicate_builder_id "$ATTESTATION" "https://github.com/slsa-framewor
 e2e_verify_predicate_builderType "$ATTESTATION" "https://github.com/slsa-framework/slsa-github-generator-go@v1"
 
 e2e_verify_predicate_invocation_configSource "$ATTESTATION" "{\"uri\":\"git+https://"github.com/$GITHUB_REPOSITORY"@$GITHUB_REF\",\"digest\":{\"sha1\":\"$GITHUB_SHA\"},\"entryPoint\":\"$GITHUB_WORKFLOW\"}"
-e2e_verify_predicate_invocation_environment "$ATTESTATION" "[\"$GITHUB_ACTOR\",\"$GITHUB_SHA\",\"ubuntu20\",\"X64\",\"$GITHUB_EVENT_NAME\",\"$GITHUB_REF\",\"$GITHUB_REF_TYPE\"]"
+
+e2e_verify_predicate_invocation_environment "$ATTESTATION" "$GITHUB_ACTOR" "github_actor"
```

Comment:
maybe it's more natural to have `"$ATTESTATION" "github_actor" "$GITHUB_ACTOR"`?

Reply:
Done.